client/components/jetpack-header/bluehost.js

```diff
@@ -0,0 +1,113 @@
+									transform="translate(108.970588 38.139706)"
+								/>
+							</g>
+							<path
+								id="Shape"
+								fill="#3575D3"
+								fillRule="nonzero"
+								d="M287.1375 58.8441176c6.538235-9.8073529 16.345588-16.3455882 28.332353-16.3455882 15.255882 0 24.518382 11.9867647 24.518382 28.8772059v49.5816177h-8.717647V73.0102941c0-16.8904412-8.717647-22.8838235-17.980147-22.8838235-11.441912 0-21.249265 8.717647-26.697794 17.4352941v53.3955883H277.875V0h8.717647v58.8441176h.544853z"
+							/>
+							<path
+								id="Shape"
+								fill="#3575D3"
+								fillRule="nonzero"
+								d="M362.327206 82.2727941c0 17.4352941 9.807353 32.1463239 28.332353 32.1463239 17.980147 0 28.332353-15.2558827 28.332353-32.1463239 0-17.4352941-10.352206-32.1463235-28.332353-32.1463235-17.980147.5448529-28.332353 15.2558823-28.332353 32.1463235zm65.927206 0c0 25.6080879-17.435294 40.3191179-37.05 40.3191179-22.338971 0-37.05-17.980147-37.05-40.3191179 0-23.4286765 16.345588-39.7742647 37.05-39.7742647 19.614706 0 37.05 14.7110294 37.05 39.7742647z"
+							/>
+							<path
+								id="Shape"
+								fill="#3575D3"
+								fillRule="nonzero"
+								d="M447.869118 107.880882c6.538235 4.358824 13.621323 6.538236 20.704411 6.538236 8.172795 0 17.980147-3.269118 17.980147-11.441912 0-7.0830884-7.627941-12.5316178-19.069852-16.8904413-13.076471-5.4485294-24.518383-10.3522059-24.518383-22.3389706 0-12.5316176 11.441912-20.7044117 26.697794-20.7044117 8.172794 0 15.255883 2.1794117 21.249265 5.4485294L487.643382 55.575c-4.903676-2.7242647-10.897058-4.3588235-17.980147-4.3588235-12.531617 0-18.525 6.5382353-18.525 13.0764706 0 8.1727941 9.2625 11.4419117 21.249265 16.3455882 18.525 7.0830882 22.338971 15.8007353 22.338971 22.8838237 0 11.986765-11.441912 19.614706-26.152942 19.614706-8.717647 0-16.890441-2.724265-24.518382-7.083089l3.813971-8.172794z"
+							/>
+							<path
+								id="Shape"
+								fill="#3575D3"
+								fillRule="nonzero"
+								d="M550.846324 43.5882353v7.0830882h-25.608089v42.4985294c0 12.5316181 5.44853 20.7044121 16.345589 21.2492651 4.903676 0 9.807352-1.089706 14.711029-3.269118l2.724265 7.627941c-4.903677 2.179412-10.897059 3.269118-16.890442 3.269118-13.621323 0-25.063235-8.717647-25.063235-28.3323531V51.2161765h-13.621323v-7.6279412h13.621323V17.4352941h8.717647v26.1529412h25.063236z"
+							/>
+						</g>
+					</g>
+					<g id="Jetpack-Logomark" fillRule="nonzero">
```

Comment:
Similar as https://github.com/Automattic/wp-calypso/pull/24830/files?utf8=%E2%9C%93&diff=unified#r189283316 -- we could use the `<JetpackLogo />` component here.